lib/tasks/import_metadata.rake

```diff
@@ -27,7 +27,10 @@ namespace :import do
       education: csv_headers[9],
       environmental_impact_assessment: csv_headers[10],
       ecosystem_assessment: csv_headers[11],
-      ecosystem_services: csv_headers[12].chomp
+      ecosystem_services: csv_headers[12],
+      license_number: csv_headers[13],
+      region: csv_headers[14],
+      country: csv_headers[15].chomp
```

Comment:
Why is `chomp` needed here and not in other cases?
Is it because of the data we are given?
If so, can we make this more consistent in any way?

Reply:
This is something that was already in place, it should assure that at the end of the last item of the header we don't include any extra space or extra special characters given by the encoding.

Reply:
Ok, makes sense. Thanks :)